src/oic/oic/provider.py

```diff
@@ -553,47 +544,118 @@ def end_session_endpoint(self, request="", cookie=None, **kwargs):
 
         logger.debug("End session request: {}".format(sanitize(esr.to_dict())))
 
-        redirect_uri = None
-        if "post_logout_redirect_uri" in esr:
-            redirect_uri = self.verify_post_logout_redirect_uri(esr, cookie)
-            if not redirect_uri:
-                msg = "Post logout redirect URI verification failed!"
-                return error_response("%s", msg)
+        # 2 ways of find out client ID and user. Either through a cookie
+        # or using the id_token_hint
+        client_id = uid = None
+        sids = []
+        cd = None
```

Comment:
`cookie_dealer` is probably better as a variable name.

Reply:
OK